client/src/api/block_builder/input_selection/core/requirement/amount.rs

```diff
@@ -147,6 +150,14 @@ impl AmountSelection {
                 *self.inputs_sdr.entry(*sdruc.return_address()).or_default() += sdruc.amount();
             }
 
+            if !input.output.is_basic() {
```

Comment:
Just to confirm: alias/foundries/nfts we want to burn are selected already earlier and then can't be added here anymore, is that true?